apps/stable/cluster-autoscaler/values.yaml.gotmpl

```diff
@@ -0,0 +1,9 @@
+{{- if eq "eks" .Values.jxRequirements.cluster.provider }}
+autoDiscovery:
+  clusterName: {{ .Values.jxRequirements.cluster.provider }}
+awsRegion: {{ .Values.jxRequirements.cluster.region }}
+rbac:
+  create: true
+  serviceAccount:
```

Comment:
Please add another provider condition around `serviceAccount`

Reply:
The current if blocks are equivalent to one big if. Please add it only around serviceAccount and it's annotations line. Leaving rbac create outside any if blocks. And please remove the line for create under serviceAccount since it's true by default

Reply:
The latest commit should be fine Gazal